perl/p5-mail-spamassassin/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           perl5 1.0
 
 perl5.branches      5.26 5.28 5.30
-perl5.setup         Mail-SpamAssassin 3.4.2
+perl5.setup         Mail-SpamAssassin 3.4.4
 revision            1
```

Comment:
The `revision` line needs to be reset to 0 when upgrading the `version`.

Reply:
Removed revision line (this seems to be the more common approach.)

Reply:
It is now preferred to leave it in with 0.